### PR TITLE
Added hamburger options in mobile menu

### PR DESCRIPTION
**Pull request by Rana-Mudassir**
- Updated The toggle Icon. Added onclick() function on it. 
- Added javascript in the index file. 
- Updated the styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,18 +20,29 @@
...
+          <nav id="layer" class="main2">
+            <div id="cross-icon">
+              <img onclick="off()" class="style-li" src="/images/close-Icon.png" alt="close-Icon">
+            </div>
+            <ul id="style">
+              <li onclick="off()" class="style-li"><a class="style-li" class='li-nav' href="#Portfolio">Portfolio</a></li>
+              <li onclick="off()" class="style-li"><a class="style-li" class='li-nav' href="#about">About</a></li>
+              <li onclick="off()" class="style-li"><a class="style-li" class='li-nav' href="#footer">Contact</a></li>
+            </ul>
+          </nav>
```
Comment (HAMMAS-SALEEM) [inline index.html:39]: ### Your Nav Menu 🔴
Student
### Figma Nav Menu 🟢
Figma

- [x] You can compare your nav menu with the figma template the first thing which you will notice that your mobile menu is missing the blur effect which you can see in the Figma template 🤔 Kindly add that to follow the Figma template 👍
- [x] You mobile menu has some horizontal scroll 😨 Kindly fix that to make your project professional 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Rana-Mudassir. @Luchy-B :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the reqested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Rana-Mudassir) [reply index.html:39]: Thanks for feeedback.

File: style.css [hunk 2]
```diff
@@ -33,6 +34,7 @@ hr {
 
 #layer {
   position: fixed;
+  backdrop-filter: blur(3px);
   height: 100%;
   top: 0;
   left: 0;
```
Comment (tugsem) [inline style.css]: - [x] Nice work adding the blur effect in the background requested by the previous reviewer 👏 . But your close-icon seems to be gone now. Check if you have the relative path in your src code to that icon. And now, page has a vertical scroll which shouldn't be there. Kindly remove that scrolling without affecting the page scrolling outside the nav menu: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hi @Rana-Mudassir , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good work applying blur effect

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Rana-Mudassir) [reply style.css]: Thanks for the feedback. Both issues are resolved now. 😃

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**